.github/workflows/e2e-tests.yml

```
@@ -0,0 +1,221 @@
+          name: playwright-report-merged
+          path: playwright-report/
+          retention-days: 30
+
+      - name: Comment PR with results
+        if: github.event_name == 'pull_request'
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const fs = require('fs');
+            const path = require('path');
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)